player_android.go

```diff
@@ -279,6 +292,19 @@ func (p *player) Write(data []uint8) (int, error) {
 }
 
 func (p *player) Close() error {
+	p.mu.Lock()
+	p.mu.Unlock()
```

Comment:
Updated (with ammend). new commit is 1e78c26 .
Mutexes are removed. Would you please review them again?
Thank you so much for your cooperation!